src/core/src/core_logic/Stopwatch.py

```diff
@@ -60,11 +62,24 @@ def stop(self):
 
         self.time_taken = self.env_layer.datetime.total_minutes_from_time_delta(self.end_time - self.start_time)
 
+    # Create new end_time even if end_time is already set
```

Comment:
why same comment here and line 54 as well? if intention is to provide information about method, change according to functionality.

Reply:
Done